src/main/java/mat/client/measure/AbstractNewMeasureView.java

```diff
@@ -278,6 +278,16 @@ protected VerticalPanel buildModelTypePanel() {
         return measureModelPanel;
     }
 
+    protected VerticalPanel buildCompositeModelTypePanel() {
+        VerticalPanel measureModelPanel = new VerticalPanel();
```

Comment:
I would avoid using Vertical/HorizontalPanels. They have some performance issues because they get translated to tables. I did the same mistake while implementing FHIR/QDM radio buttons. FlowPanel is better for this use case. we have VerticalFlowPanel(custom flow panel). Should work